saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -96,6 +97,13 @@ class OrderSettingsInput(BaseInputObjectType):
             "requested action for the transaction." + ADDED_IN_313 + PREVIEW_FEATURE
         ),
     )
+    allow_to_create_order_without_payment = graphene.Boolean(
+        required=False,
+        description=(
+            "Determine if it is possible to create order before finalizing payment "
```

Comment:
- I would mention here that we are describing here usage of `checkoutComplete` mutation (We also have orderCreateFromCheckout).  "Determine if it is possible to place unpaid order by calling `checkoutComplete` mutation"
- `TRANSACTION_FLOW` doesn't say anything. It is a value of unrelated ENUM `MarkAsPaidStrategyEnum`.  I think that we can skip this part.  Creating unpaid order is not strictly related with the payment flow that you're using. For transactionAPI you can process payment later. For paymentAPI you can mark payment as paid.